frontend/public/components/factory/table-filters.ts

```diff
@@ -26,7 +31,9 @@ export const tableFilters: TableFilterMap = {
 
   'catalog-source-name': (filter, obj) => fuzzyCaseInsensitive(filter, obj.name),
 
-  'alert-name': (filter, alert) => fuzzyCaseInsensitive(filter, _.get(alert, 'labels.alertname')),
+  'alert-list-text': (filter, alert) =>
+    fuzzyCaseInsensitive(filter, alert.labels?.alertname) ||
+    fuzzyCaseInsensitive(filter, alertDescription(alert)),
```

Comment:
Depending on how long descriptions tend to be, we might consider making this an exact text match. Fuzzy can include too many matches against longer text.

https://github.com/openshift/console/blob/master/frontend/public/components/events.jsx#L365-L379

https://bugzilla.redhat.com/show_bug.cgi?id=1603091

Reply:
Thanks for the suggestion. This was indeed reported as a bug. I think a simpler fix might work in this case: https://github.com/openshift/console/pull/4968